data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3433 @@
+				die
+			label stay
+			`	The Hai reaches for some kind of device in his pocket and enters what you assume to be the access codes. By the look on his face, it does not seem to be working very well, though... "Someone else has had access to the vault. This was not expected," he hisses as his grip tightens on his weapon and he dons his helmet. "Stay here, I will take care of it."`
+			choice
+				`	"Should you not want one extra gun, then?"`
+					goto wannafight
+				`	"What about the locals? They do not seem friendly."`
+			`	"Those lowlife scums? They would not attempt attacking a warrior, worried of the consequences," he was about to keep going but then looks at you, armorless and armed with a laser that seems like a toy in comparison to his. "Maybe you should come with me after all."`
+				goto choose
+			label outside
+			`	It may have felt like the right thing to do at the time, but you quickly realize splitting up and then staying here in the open was not very intelligent. It is too late unfortunately, because you get stabbed by a local pirate, who is no longer impressed by your now missing armored friend.`
+				die
+			label wannafight
+			`	"If the vault was compromised it was not by amateurs. I fear those inside are more dangerous, and well armed than the locals. Still I do not have to guarantee your security in any way, so you are free to do as you wish. The defense mechanisms are offline at the moment, so you would have to fend for yourself, but locals are so young and frail that even you should manage it." You hear a tone of provocation in the last words.`
+			label choose
+			choice
+				`	(Stay outside.)`
+					goto outside
+				`	(Come in.)`
+			`	He sighs, "As you wish, but I hope you can take care of yourself on the ground as well as in space. I have disabled the automated defenses just in case they have been hijacked. We just have to go in, make sure they recognize us as friends, and then reactivate them." You really wonder where that vault could be, until you notice that part of this mountain in front of you is a hologram to disguise the entry. You both enter the vault.`
+			`	At first you do not notice anything suspicious, the vault is clearly alien in origin, but you cannot pinpoint one race, instead it seems to be a mix of many. All of it is very, very old to say the least, and any lights are non functioning, probably because your friend deactivated them, but maybe even before that... At least there is quite some space to move around, enough for an armored vehicle in fact. As you get used to the darkness inside the vault, you finally notice a flash of light, quite far away on your left.`
+			choice
+				`	(It is just a light flicking, no need to worry.)`
+				`	"GET DOWN!"`
+					goto down
+			`	It was indeed just a light flicking. From a laser sight, used to acquire a target: you.`
+				die
```

Comment:
It is really meant to be a mission on the verge of suicidal for you (but you will have a checkpoint 2 jumps from here)
He is fine because he is trained and in heavy armor. You are not.

Reply:
Given that First Last probably has better gear than Si-Ra, it seems absurd to write it that way. Also, First Last has probably been in hundreds of boarding actions, maybe even against the Korath or the Pug. So s/he is definitely trained, and probably much more experienced than this hotshot squirrel kid.

Reply:
My opinion was that first last was always overseeing the boarding actions and not participating, or at least not putting himself into risk. You never die from a boarding action, except if the last crew (first last) dies. Also that kid has been trained since birth to fight.

Reply:
It was only 2 "die" but I agree with you. I mainly want to make the player understand this is a hard mission, and life with the uhai is risky. I'll make you wake up in a hospital bed